diff --git a/com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineMenuItems.cs b/com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineMenuItems.cs
index 1efeeb1c77..e6bd359ebc 100644
--- a/com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineMenuItems.cs
+++ b/com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineMenuItems.cs
@@ -300,7 +300,7 @@ namespace UnityEditor.Rendering.HighDefinition
         // [MenuItem("Edit/Render Pipeline/HD Render Pipeline/Reset All Project and Scene High Definition Materials Keywords")]
         static void ResetAllMaterialKeywordsInProjectAndScenes()
         {
-            var openedScenes = new string[SceneManager.loadedSceneCount];
+            var openedScenes = new string[SceneManager.sceneCount];
             for (var i = 0; i < openedScenes.Length; ++i)
                 openedScenes[i] = SceneManager.GetSceneAt(i).path;
 
diff --git a/com.unity.visualeffectgraph/Editor/GraphView/VFXComponentBoard.cs b/com.unity.visualeffectgraph/Editor/GraphView/VFXComponentBoard.cs
index 62de38e742..0c48764a37 100644
--- a/com.unity.visualeffectgraph/Editor/GraphView/VFXComponentBoard.cs
+++ b/com.unity.visualeffectgraph/Editor/GraphView/VFXComponentBoard.cs
@@ -616,7 +616,7 @@ namespace UnityEditor.VFX.UI
                 current = current.parent;
             }
 
-            if (UnityEngine.SceneManagement.SceneManager.loadedSceneCount > 1)
+            if (UnityEngine.SceneManagement.SceneManager.sceneCount > 1)
             {
                 path = m_AttachedComponent.gameObject.scene.name + " : " + path;
             }
